xml/System.Web.UI/IResourceUrlGenerator.xml

```diff
@@ -21,7 +21,7 @@
   
 -   On using resources in localization, see [ASP.NET Web Page Resources Overview](https://msdn.microsoft.com/library/0936b3b2-9e6e-4abe-9c06-364efef9dbbd).  
   
--   On using embedded resources, see [NIB:How to: Use Resources to Set Property Values in Web Server Controls](https://msdn.microsoft.com/library/d8e0e5c0-1958-422b-b4f5-ebe63f6c45cb) and [How to: Retrieve Resource Values Programmatically](https://msdn.microsoft.com/library/7a06d512-6fc3-40ca-9cb6-7f41d1a9fa28).  
+-   On using embedded resources, see [How to: Use Resources to Set Property Values in Web Server Controls](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-3.0/ms228093(v=vs.85)) and [How to: Retrieve Resource Values Programmatically](https://docs.microsoft.com/en-us/previous-versions/ms227982(v=vs.140)).  
```

Comment:
We remove the locale from links so that readers are directed to the correct localized version:

```suggestion
-   On using embedded resources, see [How to: Use Resources to Set Property Values in Web Server Controls](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-3.0/ms228093(v=vs.85)) and [How to: Retrieve Resource Values Programmatically](https://docs.microsoft.com/previous-versions/ms227982(v=vs.140)).  
```

Reply:
done